pantheon.md

```diff
@@ -0,0 +1,51 @@
+# Case Study: Pantheon.io/docs
+
+Pantheon is a website management platform for Drupal and Wordpress. In this use case, their Technical Content Editor Rachel Whitton describes the use of GitHub for documentation as well as their workflow.
+
+## What site or sites do you create from GitHub source? 
+https://pantheon.io/docs
+
+## What are the repositories that build the deliverables? 
+https://github.com/pantheon-systems/documentation 
+
+## Can you give an idea of the size, such as how many pages (source and output)? If they're private, that's fine too, talk about what led to that decision.
+Source files: 588 These are all publicly-editable source files.
+Output files: 602
+
+## What factors led your team to choose GitHub for docs? Why and when was it chosen?
+
```

Comment:
These should probably be two questions, can you answer the "when" portion -- and also when was the migration?